lib/SILOptimizer/Mandatory/TFDeabstraction.cpp

```diff
@@ -52,6 +52,10 @@ static llvm::cl::opt<bool> TFPromoteGlobalVariables(
         "If enabled, promote global variables into SSA with a best "
         "effort to minimize sends/recvs. This is a performance optimization."));
 
+namespace llvm {
+  extern cl::opt<bool> TFDynamicCompilation;
```

Comment:
Is there a header file (in SILOpt/Mandatory) this can live in?

Reply:
TFUtilities.h makes sense maybe? It already has other flag-ish things (`getTFDumpIntermediateStream`) that all the TF* files use. I moved it there, and also moved the definition to TFUtilities.cpp.